addons/base_import/static/tests/import_buttons_tests.js

```diff
@@ -107,50 +106,4 @@ QUnit.test('kanban without import button', function(assert) {
     kanban.destroy();
 });
 
-QUnit.test('import button should be hidden in list on mobile screens', function (assert) {
-    assert.expect(1);
-
-    var list = createView({
-        View: ListView,
-        model: 'foo',
-        data: this.data,
-        arch: '<tree><field name="foo"/></tree>',
-        config: {
-            device: {
-                size_class: config.device.SIZES.XS,
```

Comment:
I hesitate to force size_class in https://github.com/odoo-dev/odoo/blame/12.0-mobile-tests-fix-adr/addons/web/views/webclient_templates.xml#L585

Reply:
no need actually, as the code only checks the isMobile property

Reply:
forget what i said, i agree that it should be forced as well as isMobile